net/ipv4/ip_output.c

```diff
@@ -701,8 +701,40 @@ struct sk_buff *ip_frag_next(struct sk_buff *skb, struct ip_frag_state *state)
 		len &= ~7;
```

Comment:
It looks like we have very similar code in `ip6_frag_next()` - why don't we patch the IPv6 code as well?